### ISSUE TITLE
Possibility to group by string value in StringColumn

### ISSUE THREAD

**Issue by puehringer** [lineupjs/lineupjs#192]
Currently, it is not possible to group a `StringColumn` by the actual value of the string (only `starts with ...` and `RegExp ...` are supported). 
With identical strings, it sometimes does not make sense to make it a `CategoricalColumn`, but grouping should be possible nonetheless.

**Comment by sgratzl** [comment]
From a conceptional point of view a StringColumn contains almost unique values. Thus, enabling this feature will create in the worst case 1 group per row. I implemented the `startsWith` and `regex..` grouping as way to avoid creating too many groups

**Comment by dvmichaelpuehringer** [comment]
We currently have the problem that sometimes our ranking contains duplicate values in a StringColumn. These duplicate values should be marked as group, allowing us to visualize the grouped box-plot for example. We have found no other way of merging two rows with a common value in this column, except for creating a group for each of them. We also found no significant performance hit and also set the default to the original default (starts with...) to avoid confusion. 
Is there a specific reason to avoid 1 group per row (only in the worst case)?
